.github/workflows/ci.yml

```
@@ -2,50 +2,42 @@ name: CI
     branches: [ main ]
 
 jobs:
-  build-and-test:
+  test:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [18.x, 20.x]
-
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)